package.json

```diff
@@ -1,27 +1,35 @@
 {
```

Comment:
Could you restore 2 space indentation, as it was before. Too many changes otherwise.